src/test/java/picard/illumina/IlluminaBasecallsToSamTest.java

```diff
@@ -65,9 +65,9 @@ public String getCommandLineProgramName() {
 
     @Test
     public void testTileNumberComparator() {
```

Comment:
Can you add a test for the sorted and non-sorted cases - as you have for IlluminaBasecallsToFastq?